CHANGES.md

```diff
@@ -1,3 +1,9 @@
+next
+----
+
+- Fix an `exception End_of_file` crash that could happen on Windows
```

Comment:
I don't think this change is observable to users because the incorrect reads were done in the cram execution. And cram hasn't been released yet.